lib/PhasarLLVM/Pointer/FilteredLLVMAliasSet.cpp

```diff
@@ -0,0 +1,259 @@
+#include "phasar/PhasarLLVM/Pointer/FilteredLLVMAliasSet.h"
+
+#include "phasar/PhasarLLVM/Pointer/LLVMAliasInfo.h"
+#include "phasar/PhasarLLVM/Pointer/LLVMAliasSet.h"
+#include "phasar/PhasarLLVM/Pointer/LLVMPointsToUtils.h"
+#include "phasar/PhasarLLVM/Utils/LLVMShorthands.h"
+#include "phasar/Pointer/AliasResult.h"
+#include "phasar/Utils/NlohmannLogging.h"
+
+#include "llvm/IR/Instructions.h"
+
+#include "nlohmann/json_fwd.hpp"
+
+#include <type_traits>
+
+using namespace psr;
+
+static const llvm::Function *getFunction(const llvm::Value *V) {
+  if (const auto *Inst = llvm::dyn_cast<llvm::Instruction>(V)) {
+    return Inst->getFunction();
+  }
+  if (const auto *Arg = llvm::dyn_cast<llvm::Argument>(V)) {
+    return Arg->getParent();
+  }
+  return nullptr;
+}
+
+[[nodiscard]] static bool isConstantGlob(const llvm::GlobalValue *GlobV) {
```

Comment:
```suggestion
[[nodiscard]] static bool isConstantGlobalValue(const llvm::GlobalValue *GlobV) {
```
I find Glob not super easy to understand. I immediatly mixed it with shell globbing.

Reply:
done